src/Card/Card.js

```diff
@@ -7,11 +7,12 @@ import { themeGet, createComponent } from '../utils';
 const StyledCard = createComponent({
   name: 'Card',
   as: Box,
-  style: ({ shadow }) => css`
+  style: ({ shadow, theme }) => css`
```

Comment:
Let's default shadow to false now - I don't think we're using shadowed cards very often